src/models/mock-metrics.ts

```diff
@@ -285,14 +285,14 @@ const aggregationResponseList: MetricsTableResponse[] =  [{
 }];
 
 
-export const computableResponseMock: ComputableResponse = {
-    columns: [
-        {collection: 'toto', metric: ArlasApiMetric.CollectFctEnum.AVG, field: 'metadata_ObservationContext_eo_opt_cloudCoverPercentage'},
-        {collection: 'toto', metric: ArlasApiMetric.CollectFctEnum.SUM, field: 'metadata_ObservationContext_eo_opt_cloudCoverPercentage'},
-        {collection: 'titi', metric: ArlasApiMetric.CollectFctEnum.AVG, field: 'metadata_ObservationContext_eo_opt_cloudCoverPercentage'},
-        {collection: 'titi', metric: ArlasApiMetric.CollectFctEnum.AVG, field: 'metadata_tristesse'},
-        {collection: 'titi', metric: ArlasApiMetric.CollectFctEnum.SUM, field: 'metadata_heureux'},
-        {collection: 'titi', metric: 'count', field: ''} // quizz du field ??? comment on sait que c'est le bon count
-    ],
-    metricsResponse: aggregationResponseList
-};
+// export const computableResponseMock: ComputableResponse = {
+//     columns: [
+//         {collection: 'toto', metric: ArlasApiMetric.CollectFctEnum.AVG, field: 'metadata_ObservationContext_eo_opt_cloudCoverPercentage'},
+//         {collection: 'toto', metric: ArlasApiMetric.CollectFctEnum.SUM, field: 'metadata_ObservationContext_eo_opt_cloudCoverPercentage'},
+//         {collection: 'titi', metric: ArlasApiMetric.CollectFctEnum.AVG, field: 'metadata_ObservationContext_eo_opt_cloudCoverPercentage'},
+//         {collection: 'titi', metric: ArlasApiMetric.CollectFctEnum.AVG, field: 'metadata_tristesse'},
+//         {collection: 'titi', metric: ArlasApiMetric.CollectFctEnum.SUM, field: 'metadata_heureux'},
+//         {collection: 'titi', metric: 'count', field: ''} // quizz du field ??? comment on sait que c'est le bon count
+//     ],
+//     metricsResponse: aggregationResponseList
+// };
```

Comment:
you can remove thus part. It was only for test

Reply:
I'll remove it in the final pr